PayloadPkg/FirmwareUpdate/FirmwareUpdateHelper.c

```diff
@@ -1033,6 +1036,73 @@ CheckAcmSvn (
   return EFI_SUCCESS;
 }
 
+#if defined (PLATFORM_RPLA)
+  /**
+    Get cpu ucode revision.
+
+    @retval   cpu ucode revision
+  **/
+  UINT32
+  EFIAPI
```

Comment:
EFIAPI is not necessary for local function.

Reply:
The function was already defined in SocInfoLib.h, I just added the implementation.